src/components/badges/_badges.scss

```diff
@@ -15,7 +14,7 @@ $include-html: false !default;
     border-radius: $badgeFontSize;
     line-height: $badgeFontSize;
     font-size: $badgeFontSize;
-    font-family: $badgeFontFamily;
+    font-weight: bold;
```

Comment:
We haven't used `bold` anywhere AFAIK and we shouldn't do that - we have a separate family (ExtraBold) for that.